pkg/operation/seed/seed.go

```diff
@@ -728,24 +724,29 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 	// Deploy component specific resources
 	for _, componentFn := range []component.BootstrapSeed{
 		clusterautoscaler.BootstrapSeed,
+		func(ctx context.Context, c client.Client, namespace, version string) error {
```

Comment:
can this function also be moved to the component specific directory like for the cluster autoscaler? 
I imagine this become quite hard to read with multiple in-line functions for Seed Bootstrapping.

Reply:
No, otherwise, this for loop here won't work anymore. I already had a chat with @timuthy and agreed that we will componentify also these seed-specific deploy/destroy functions, but I will do it in a separate PR once this PR and #3133 are merged.